### PR TITLE
docs(*) missing docs for 2.2 release

### PR DESCRIPTION
**Pull request by locao**
- Updated hybrid-mode docs with new data_planes path
- Update correlation-id plugin version
- Added grpc-web new config param
- Added new possible value to limit_by config param

### CODE REVIEW

File: app/_hub/kong-inc/correlation-id/1.0.0.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
+---
+name: Correlation ID
+publisher: Kong Inc.
+version: 1.0.0
```
Comment (tlblessing) [inline app/_hub/kong-inc/correlation-id/1.0.0.md:4]:
```suggestion
version: 1.0.x
```
File needs to be renamed to 1.0.x and a 2.2.x version needs to be added to the yaml file. @locao what changed for 2.2? Just supported by release?

File: app/_hub/kong-inc/correlation-id/index.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 name: Correlation ID
 publisher: Kong Inc.
-version: 1.0.0
+version: 2.0.2
```
Comment (tlblessing) [inline app/_hub/kong-inc/correlation-id/index.md:4]:
```suggestion
version: 2.0.x
```

**Review by tlblessing — CHANGES_REQUESTED** [review]
I'm going to check out this branch and make some changes to plugin versions files.

**Comment by tlblessing** [comment]
Direct review links:

Correlation ID plugin

File: app/_hub/kong-inc/correlation-id/1.0.0.md (hunk 1 above)
Comment (locao) [reply app/_hub/kong-inc/correlation-id/1.0.0.md]: I noticed that this plugin had its version bumped from 1.0 to 2.0 a long time ago (May 2019) but we didn't update the docs accordingly. There are no other relevant changes we should add to its docs.

**Comment by locao** [comment]
I updated the version in the gRPC web plugin doc to match the plugin's internal version, I guess that's how we are doing in the other ones, right?

**Comment by tlblessing** [comment]
> I updated the version in the gRPC web plugin doc to match the plugin's internal version, I guess that's how we are doing in the other ones, right?

@locao thank you. I guess its versioning has been incorrect for some time. Some plugins follow internal version.

**Review by tlblessing — APPROVED** [review]
Thank you @locao !

**Pull request merged**